cloud-azure/src/test/java/com/sequenceiq/cloudbreak/cloud/azure/AzureEncryptionResourcesTest.java

```diff
@@ -139,6 +142,40 @@ private void initActionFailedExceptionConversion() {
                 .thenAnswer(invocation -> new Retry.ActionFailedException(new CloudConnectorException(invocation.getArgument(0, Exception.class))));
     }
 
+    private static Object[][] getDesResourceId() {
```

Comment:
As a convention, such methods shall be named so their purpose is clear. If used for a single test method, it makes sense to name them as _testMethod_`DataProvider`. So in this case something like `testDiskEncryptionSetResourceGroupNameExtractionDataProvider` would sound more appropriate.

Note: `DataProvider` is a TestNG terminology; it was the first test library that provided support for parameterized tests, way before JUnit.